docs/topics/compiler-reference.md

```diff
@@ -144,6 +144,14 @@ Suppresses specific warnings [globally across the whole project](whatsnew21.md#g
 kotlinc -Xsuppress-warning=NOTHING_TO_INLINE -Xsuppress-warning=NO_TAIL_CALLS_FOUND main.kt
 ```
 
+### -Xrepl
+
+Activates the Kotlin REPL to have an interactive shell.
```

Comment:
Here I'm touching language related things that you definitely know better than me, but I feel like the phrase is unnecessarily recursive. REPL as a term stands for an interactive shell. Can we just drop the rest of the sentence, starting with "to have"? Or formulate it differntly?

Reply:
Sure! I shortened it to "Activates the Kotlin REPL."